core-java/src/main/java/com/baeldung/algorithms/annealing/City.java

```diff
@@ -2,21 +2,35 @@
 
 import lombok.Data;
 
-@Data
-public class City {
+@Data public class City {
```

Comment:
No but this class had an issue was not able to build the project without the minor changes i added